script_library/deployment-actions-common.sh

```diff
@@ -76,5 +78,13 @@ function clean_userfiles(){
 }
 function gather_logs(){
     echo "Gathering kubernetes logs"
+    setup_ssh_agent
```

Comment:
this should be not needed

Reply:
See so this ssh-agent stuff was done for CI only and it should stay CI only. Introducing it in a normal run.sh has consequences and it may end up in the final product somehow...We dont want that, this is a hack to make it work for CI otherwise this should be in the docs like its on the skuba repo: https://github.com/SUSE/skuba#prerequisites

We should not be playing with those things unless we control where its gonna be run. I sure dont want to support this at all. :)

Reply:
In recent PR updates, I have removed ssh-agent part of logic to keep it moving.